### DISCUSSION TITLE
Cannot download on Mac

### DISCUSSION THREAD

**Discussion by newsherlytemple** [blue-yonder/tsfresh#950] [Q&A] [answered]
Hello,

Whenever I try to install using "pip install tsfresh", I get one of two errors, either it cannot build the dependencies, or it cannot generate package metadata. I'm not sure if there's something specific you need to do for Mac BigSur? I am new to the MacOS.

Thank you

**Comment by DvdKt** [comment]
I have the same problem!

**Comment by nils-braun** [comment, marked as answer]
tsfresh version 0.20.0 removed an incompatible dependency, so you should now be able to install it on macOS again. 
If you still face issues, place tell us :)
Thank you for your patience!
